frontend/src/templates/Field/Table/TableField.tsx

```diff
@@ -135,6 +135,13 @@ export const TableField = ({
     return description
   }, [fields.length, schema.addMoreRows, schema.maximumRows])
 
+  // If a table is the last field, Chrome and MS Edge sometimes truncate the table in print mode.
```

Comment:
From our experiments, looks like height for all tables is not computed correctly, so the comment should be updated to remove `is the last field`

Reply:
[023dee2](https://github.com/opengovsg/FormSG/pull/5596/commits/023dee2d290577a3d91eeb0cedc91ef2939b9970)